docs/UserGuide.md

```diff
@@ -380,13 +381,37 @@ Format: `sort [sna/] [snd/] [ssa/] [ssd/] [so/]…​`
 
 ### Importing a show: `import`
 
+**Description:** Want to easily import Trackermon data from other devices? Just use our import function!
+
+**Format:** `import`
+
+**Example & Output:** `import`
+- Step 1. Navigate to the Trackermon data you want to import using your OS' native GUI!
+- Step 2. Select the file and click "Open"
+  - Do note that only JSON files can be imported!
+- Step 3. Trackermon will import the selected file and display the imported show list!
+  - If there was an error importing the file, Trackermon will display an error message!
+
+<img src="images/Import.png">
+
 
 [return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
 ---
 
 ### Exporting a show: `export`
 
+**Description:** Want to easily export Trackermon data to other devices? Just use our export function!
+
+**Format:** `export`
+
+**Example & Output:** `export`
+- Step 1. Navigate to the location you wish to export Trackermon data to using your OS' native GUI!
```

Comment:
Maybe you could link GUI to the glossary? 🤔

Reply:
Nice catch! I'll update the UG.